conf/mini_provisioner/v2/ha.cleanup.tmpl

```diff
@@ -0,0 +1,22 @@
+cluster:
```

Comment:
1. It would be good to create ha.cleanup.tmpl.1-node and ha.cleanup.tmpl.3-node. Some of the entries will replicate.
2. Would be good to provide sample files (optional), e.g. ha.cleanup.tmpl.3-node.sample

Reply:
@ujjwalpl  Done. The templates for 1-node and 3-nodes are same now. We would update the 3-nodes ones as we implement 3-node support through next sprint.